README.md

```diff
@@ -53,25 +53,27 @@ Every newly submitted transaction should go on **the top of the list** and shoul
 
 ### If you run into a problem
 
-Need help? Head over to [our community on GitHub](https://github.com/orgs/DevSkillsHQ/discussions/categories/help) to get assistance.
+Head over to [our community on GitHub](https://github.com/orgs/DevSkillsHQ/discussions/categories/help) to get assistance.
 
-### Import boilerplate
+### Import a boilerplate project
 
-Use [this link](https://docs.devskills.co/collections/85-the-interview-process/articles/342-importing-challenge-boilerplate) to get boilerplate code for your tech stack to configure a minimal setup for running the E2E tests.
+We have created a set of boilerplate projects for different tech stacks to help you get started quicker.
+
+To import a boilerplate project:
+
+1. Check out [this list](https://help.alvalabs.io/en/articles/7972852-supported-coding-test-boilerplates) to pick a desired boilerplate and copy its name (e.g., `backend-boilerplate-php-laravel`).
+2. Go to the "Actions" tab of your GitHub repository and select the "Setup boilerplate" workflow in the left side panel.
+3. In the "Run workflow" dropdown, paste the previously copied boilerplate name along with the branch name where you want the boilerplate to be imported (e.g., `implementation`) and click the "Run workflow" button.
+4. After the workflow has finished, your selected boilerplate will be imported to the specified branch, and you can continue with your task there.
 
 <details>
-<summary>Alternatevily, use the manual setup.</summary>
+<summary>If you instead want to use a custom setup, complete the steps below to make the E2E tests run correctly.</summary>
 
 1. Update the `baseUrl` (where your app will run) in [cypress.json](cypress.json).
 2. Update the [`build`](package.json#L5) and [`start`](package.json#L6) scripts in [package.json](package.json) to respectively build and start your app.
 
 </details>
 
-### Get familiar with the API
-
-<details>
-<summary>Request examples</summary>
-
 ##### Get historical transactions
```

Comment:
@fenske,
What should be done to the API endpoints related to account transactions mentioned below?

Reply:
No, this part we should leave as is as frontenders don't get the OpenAPI spec.